app/models/miq_alert_status_state.rb

```diff
@@ -0,0 +1,7 @@
+class MiqAlertStatusState < ApplicationRecord
```

Comment:
@imtayadeway the whole naming of the alerts is confusing, miq-alert is actually an alert_definition, miq-alert-status is an instance of that alert_definition, and miq-alert-status-state is an action/state that is made on that alert (being commented/acked/assigned).
I could rename state with action if you think its better.